api/src/services/users.ts

```diff
@@ -98,6 +99,18 @@ export class UsersService extends ItemsService {
 		}
 	}
 
+	private validateEmails(emails: string[]) {
+		const schema = Joi.string().email().required();
+
+		for (const email of emails) {
+			const { error } = schema.validate(email);
+
+			if (error) {
+				throw new FailedValidationError(joiValidationErrorItemToErrorExtensions(error.details![0]!));
```

Comment:
It seems to be something wrong with the parameter here 🤔 when I submit an invalid email address while creating a user, I now receive an error saying "Couldn't extract validation error type from Joi validation error item"

Reply:
Yeah, it seems like `joiValidationErrorItemToErrorExtensions` is missing the `string` case:
```js
details: [
  {
    message: '"value" must be a valid email',
    path: [],
    type: 'string.email',
    context: [Object]
  }
]
```

Reply:
I agree that `FailedValidationError` should probably fallback to another message on unhandled case in `joiValidationErrorItemToErrorExtensions`.